provisioning/transport/amqp/src/SaslTpmHandler.cs

```diff
@@ -61,8 +60,7 @@ public override int GetHashCode()
             {
                 var hashCode = _endorsementKey != null ? _endorsementKey.GetHashCode() : 0;
```

Comment:
Please add the type if not present on the same line.
i.e. var myVar = new MyType() is OK because the type is written in the same statement.